README.md

```diff
@@ -51,7 +67,7 @@ Because the subsampling and zero-removal stages involve pseudo-random sample rem
 The results can be reproduced by providing the same inputs and date from a previous experiment.
 
 ## Example
-`SCINGE_Example.m` demonstrates a simple example with two hyperparameter sets and two replicates.
```

Comment:
Should we move this to the `MATLAB environment` subsection above?  That way we discuss each mode of execution and then immediately follow it with the example of how to run using that mode.